blockly-core/core/ui/block.js

```diff
@@ -254,14 +254,14 @@ Blockly.Block.isFreelyDragging = function() {
 
 /**
  * Wrapper function called when a mouseUp occurs during a drag operation.
- * @type {Array.<!Array>}
+ * @type {bindData}
```

Comment:
I'll drop this thread in HipChat to give the whole team visibility on this.

I'm in favor of a quick refactor to PascalCase types across our apps code.